.github/workflows/build.yml

```diff
@@ -200,21 +198,21 @@ jobs:
 
       - name: Upload Windows Installer
         run: |
-          gh release upload ${{ github.ref_name }} NotepadNext-Windows-Qt5.15.2-Installer/NotepadNext-${{ github.ref_name }}-Installer.exe
+          gh release upload ${{ github.ref_name }} NotepadNext-Windows-Qt6.2.4-Installer/NotepadNext-${{ github.ref_name }}-Installer.exe
 
       # The artifact is unzipped, so need to zip it back up so it can be uploaded
       - name: Upload Windows Zip
         run: |
-          cd NotepadNext-Windows-Qt5.15.2-Zip/
+          cd NotepadNext-Windows-Qt6.2.4-Zip/
```

Comment:
Just call it `Qt6` - this way the next time you upgrade Qt, less to modify.

Reply:
The output directories are based on the `matrix.config.qt_version` variable so it has the full version number in there. Plus if there is a 6.2.4 and 6.3.1 then they'd have to be different. Something like string splitting to have `Qt6.2` and `Qt6.3` might be a viable option though.